### DISCUSSION TITLE
Bài tập 19/11

### DISCUSSION THREAD

**Discussion by phucthinhb3** [phucthinhb3/Lop-11A03-Nhom-07#8] [General]
-Phân chia làm bài tập đã giao trên notion nha:
+Ken với tao thì viết code cho
+Phong,Tín,Khôi thì nối mạch nhá
-Làm xong nhớ để vô project nhá

**Comment by tintruongvinh** [comment]
Ok hiểu rồi

**Comment by tuananh-b3-03** [comment]
okk

**Reply by trinhtienphong** [reply]
okk

**Comment by phucthinhb3** [comment]
@HienCaoPhuNhuan Dạ thưa thầy cái bài tập lắp mạch đèn giao thông ấy mình có phải tạo project rồi thêm các dữ liệu vô đó không ạ? Ví dụ như code và ảnh lắp mạch ấy thầy

**Reply by HienCaoPhuNhuan** [reply]
Mình phân tích ra các bước của code nó sẽ làm gì làm gì

**Reply by HienCaoPhuNhuan** [reply]
Nghĩa là đọc - hiểu code đó

**Reply by phucthinhb3** [reply]
@HienCaoPhuNhuan  Dạ con hiểu ạ con cám ơn thầy.  Mà thầy ơi cái thông báo mới nhất thầy đăng trên notion ấy thầy có nhắc đến bài tập 2,3,4 . Dạ bài tập đó là những bài nào vậy thầy?

**Reply by HienCaoPhuNhuan** [reply]
> @HienCaoPhuNhuan  Dạ con hiểu ạ con cám ơn thầy.  Mà thầy ơi cái thông báo mới nhất thầy đăng trên notion ấy thầy có nhắc đến bài tập 2,3,4 . Dạ bài tập đó là những bài nào vậy thầy?  

Là bài 7 hôm nay đó, mở ra

**Reply by phucthinhb3** [reply]
Dạ con cám ơn thầy ạ
